src/map/pc.c

```diff
@@ -9986,51 +9986,51 @@ void pc_unequipitem_pos(struct map_session_data *sd, int n, int pos)
  *------------------------------------------*/
 int pc_unequipitem(struct map_session_data *sd,int n,int flag)
 {
-	int i,iflag;
-	bool status_cacl = false;
+	int i, iflag;
+	bool status_calc = false;
 	int pos;
+
 	nullpo_ret(sd);
 
-	if( n < 0 || n >= MAX_INVENTORY ) {
-		clif->unequipitemack(sd,0,0,UIA_FAIL);
+	if (n < 0 || n >= MAX_INVENTORY) {
+		clif->unequipitemack(sd, 0, 0, UIA_FAIL);
 		return 0;
 	}
 
 	// if player is berserk then cannot unequip
-	if (!(flag&PCUNEQUIPITEM_FORCE) && sd->sc.count && (sd->sc.data[SC_BERSERK] || sd->sc.data[SC_NO_SWITCH_EQUIP]) )
-	{
-		clif->unequipitemack(sd,n,0,UIA_FAIL);
+	if (!(flag&PCUNEQUIPITEM_FORCE) && sd->sc.count && (sd->sc.data[SC_BERSERK] || sd->sc.data[SC_NO_SWITCH_EQUIP])) {
```

Comment:
if you changing style on this line, probably also add missing spaces after/before &